### DISCUSSION TITLE
Autocomplete component with screen reader.

### DISCUSSION THREAD

**Discussion by chschrodahl** [primefaces/community#1574] [PrimeVue]
This component works well with screen readers. 
Just something i noticed that i don't know if it is normal or not.
It kind of make sense but at the same time not.

When a user has screen reader enabled it will tell you everything that is focused.
As you can see is that it works very good. but when you select an item it will start telling the user that:
No result Found. 
Combobox collapsed.
Has Auto complete editable Canada.

I have not used screen reader to much. That's why i ask.
Should it ignore the No Result Found and maybe some of the other states it tells the user? 
And/Or change the order of the what it tells the user (If possible?) 

Another question if someone know.
How would this work if we add sub categories inside here? like: Autocomplete with accordion inside ?

https://github.com/primefaces/community/assets/158478689/1561a743-e9a4-4ef9-9b79-04ec8525dabc

**Comment by jennylgh** [comment]
Ditto [chschrodahl](https://github.com/chschrodahl). I ran into same issue when testing Mac VoiceOver and Windows NVDA. Screenread will announce "No results found" on different occasions when it's not necessary. 
For example, user is just navigating around page content and it passes through the AutoComplete component. 
Is it possible to read out label "No results found" only when popup panel is open? or probably combine with some other conditions to refine or restrict when it's being announced by screenreader.
